addons/sourcemod/scripting/TogsJumpStats.sp

```diff
@@ -853,6 +846,9 @@ void GetClientStats(int client, char[] sStats, int iLength)
 
 public Action OnPlayerRunCmd(int client, int &buttons, int &impulse, float a_fVel[3], float a_fAngles[3], int &weapon)
 {
+	if(!IsClientInGame(client))
+		return Plugin_Handled;

```

Comment:
Plugin_Continue

Reply:
No. If client isn't ingame, why continue ? will return an errror.

- L 08/02/2022 - 09:57:47: [SM] Exception reported: Client 21 is not in game
- L 08/02/2022 - 09:57:47: [SM] Blaming: custom/TogsJumpStats.smx
- L 08/02/2022 - 09:57:47: [SM] Call stack trace:
- L 08/02/2022 - 09:57:47: [SM]   [0] IsPlayerAlive
- L 08/02/2022 - 09:57:47: [SM]   [1] Line 847 TogsJumpStats/addons/sourcemod/scripting/TogsJumpStats.sp::OnPlayerRunCmd

Reply:
![image](https://user-images.githubusercontent.com/19607336/185338270-7dbed783-771f-4d70-a874-28c997a250c2.png)
Its not what you think, if you return Plugin_Handled, it will block all other plugins that have OnPlayerRunCmd